LICENSE

```diff
@@ -1,16 +1,24 @@
+The MIT License
+
+Copyright (c) 2007-2019 Dinesh Manajipet <dinesh.manajipet@proemion.com>
```

Comment:
:question: Has been the license changed even on the upstream mongoose?

Reply:
Oh there are 2 licenses. One is the license of upstream mongoose C library - which is GPL something and commercial something.

Then there is the license of the mongoose-cpp fork we started off with. That was a "fork" of mongoose C library - but the license is MIT. With this pull request - the upstream mongoose is just a git submodule inside vendor/ , and we rewrote so much of the mongoose-cpp library - thought to update the license.